compiler-user-guide/src/general/faqs.md

```diff
@@ -38,7 +38,7 @@
 
 - **Q**: Will Clash work with my EDA tools?
 
-  **A**: In general, Clash should work well with Xilinx and Intel FPAGs and their EDA tools -- as development typically focuses on these vendors.
+  **A**: In general, Clash should work well with Xilinx and Intel FPAGs and their EDA tools &mdash; as development typically focuses on these vendors.
```

Comment:
Oh doesn't mdBook handle this? That's sad :(

Reply:
You _can_: https://rust-lang.github.io/mdBook/format/markdown.html#smart-punctuation

But that enables a host of other replacements as well. I didn't know if you deliberately not enabled that optional setting, but I thought the few places we want an em dash could just be done explicitly. By the way, `--` would be replaced with an en dash, so it should have been changed to `---` even if you enable "smart punctuation".